### PR TITLE
[PCA] move computeCorrelationEigenvalues from CPU to GPU 

### PR DESCRIPTION
**Pull request by mchernov-intel**

### CODE REVIEW

**Comment by mchernov-intel** [comment]
@PetrovKP @mchernov-intel please, review

**Comment by mchernov-intel** [comment]
make test

File: include/oneapi/internal/math/lapack_executor.h [hunk 1]
```diff
@@ -143,11 +143,69 @@ class PotrsExecutor
...
+        template <typename T>
+        void operator()(Typelist<T>)
+        {
+            auto a_buffer_t = a.template get<T>();
+            auto w_buffer_t = w.template get<T>();
+            auto work_buffer_t = work.template get<T>();
+            auto iwork_buffer_t = iwork.template get<int64_t>();
+
+#ifdef ONEAPI_DAAL_NO_MKL_GPU_FUNC
+            // TODO
```
Comment (PetrovKP) [inline include/oneapi/internal/math/lapack_executor.h]: fix todo!

**Comment by mchernov-intel** [comment]
make test

**Comment by mchernov-intel** [comment]
make test

**Comment by mchernov-intel** [comment]
make test

**Comment by mchernov-intel** [comment]
make test

**Pull request closed without merging**